lib/spotlight/dor/indexer.rb

```diff
@@ -50,6 +52,12 @@ def add_box(sdb, solr_doc)
         solr_doc['box_ssi'] = sdb.smods_rec.box
       end
 
+      # pulls from //subject/geographic
+      def add_geonames(sdb, solr_doc)
+        ids = extract_geonames_ids(sdb)
+        solr_doc['geographic_srpt'] = ids.map { |id| get_geonames_api_envelope(id) }.reject(&:nil?)
```

Comment:
Here, you are exercising your preference in a truly arbitrary way over trivialities.  This distracts from larger gestalt problems like _this gem shouldn't even exist_.  You should apply a reasonable standard of acceptability, not stylistic exactitude even **beyond rubocop**.  We do not have *exactly* the same style and we don't have to.  Where no rule is formalized, you should anticipate and allow some variation.  You understand what I wrote perfectly well.  The performance is not an actual issue.  This is below the threshold of what should even prompt discussion.  The individual changes: Yeah, I can live with that.  The arbitrary exactitude: No, I can't.